e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v1_test.js

```diff
@@ -3,6 +3,7 @@
 const config = require('../../../config.js');
 const mpScenario = 'ONE_V_ONE';
 const caseWorkerUser = config.hearingCenterAdminWithRegionId1;
```

Comment:
I would suggest using user for "caseWorkerUser" , "JudgeUser" if EA courts are enabled or not

Reply:
Done